cmd/workflow-controller/main.go

```diff
@@ -23,6 +21,7 @@ import (
 )
 
 const (
+	// CLIName is the name of the CLI
```

Comment:
Any reason the cli is called "workflow-controller"?

Reply:
Just wanted to match the container image name and deployment name. I believe ingress-controller is the same way.